packages/frontend/src/ee/features/aiCopilot/components/AiAgents.tsx

```diff
@@ -1,128 +1,79 @@
-
-const formSchema = z.object({
-    notificationChannel: z.string().min(1).nullable(),
-    appProfilePhotoUrl: z.string().url().nullable(),
-    slackChannelProjectMappings: z.array(
-        z.object({
-            projectUuid: z
-                .string({ message: 'You must select a project' })
-                .uuid({ message: 'Invalid project' }),
-            slackChannelId: z
-                .string({
-                    message: 'You must select a Slack channel',
-                })
-                .min(1),
-            availableTags: z.array(z.string().min(1)).nullable(),
-        }),
-    ),
-});
 
 export const AiAgents: FC = () => {
+    const navigate = useNavigate();
     const { data: slackInstallation } = useGetSlack();
     const organizationHasSlack = !!slackInstallation?.organizationUuid;
 
     const { data: slackChannels } = useSlackChannels('', true, {
         enabled: organizationHasSlack,
     });
 
-    const { mutateAsync: updateCustomSettings, isLoading: isSubmitting } =
-        useUpdateSlackAppCustomSettingsMutation();
-
-    const form = useForm<SlackAppCustomSettings>({
-        initialValues: {
-            notificationChannel: null,
-            appProfilePhotoUrl: null,
-            slackChannelProjectMappings: [],
-        },
-        validate: zodResolver(formSchema),
-    });
-
-    useEffect(() => {
-        if (!slackInstallation) return;
-
-        const initialValues = {
-            notificationChannel: slackInstallation.notificationChannel ?? null,
-            appProfilePhotoUrl: slackInstallation.appProfilePhotoUrl ?? null,
-            slackChannelProjectMappings:
-                slackInstallation.slackChannelProjectMappings ?? [],
-        };
-
-        if (form.initialized) {
-            form.setInitialValues(initialValues);
-            form.setValues(initialValues);
-        } else {
-            form.initialize(initialValues);
-        }
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [slackInstallation]);
-
-    const slackChannelOptions = useMemo(() => {
-        return (
-            slackChannels?.map((channel) => ({
-                value: channel.id,
-                label: channel.name,
-            })) ?? []
-        );
-    }, [slackChannels]);
-
     const { data: projects } = useProjects();
 
-    const projectOptions = useMemo(() => {
-        return (
-            projects?.map((project) => ({
-                value: project.projectUuid,
-                label: project.name,
-            })) ?? []
-        );
-    }, [projects]);
-
-    const { refresh: refreshChannels, isLoading: isRefreshing } =
-        useSlackChannels('');
-
-    const handleSubmit = form.onSubmit(async (args) => {
-        if (organizationHasSlack) {
-            await updateCustomSettings(args);
+    const agentList = useMemo(() => {
+        if (!slackInstallation?.slackChannelProjectMappings?.length) {
+            return undefined;
```

Comment:
```suggestion
            return null;
```

just a nit, but I know this might end up in philosophical discussion 😅